src/molecules/Modal/modal.module.scss

```diff
@@ -24,6 +24,14 @@ $border: 1px solid color('neutral-5');
       padding: 0;
       border-bottom: none;
     }
+
+    .section:not( :first-child ) {
```

Comment:
@jmcolella Unless I'm missing something, you could have just updated the `.section` props, and then have `:first-child` override?